### PR TITLE
Module 1: Awesome Books Library navigation

### PR DESCRIPTION
**Pull request by the-catalystmc**
In this project we:

 Modify the Awesome books application to have:
- A Navigation bar.
- Three content sections:
- Books list.
- Add book form.
- Contact info.

- Display the current date below the navbar using Luxon.
- Used plain Javascript CSS and HTML.
- Make sure that when the user clicks on any of the links, the main area changes to display only one of the three content sections.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -69,6 +69,40 @@ const library = new Library([]);
...
 const addBtn = document.querySelector('.add__add-btn');
 
+/* global luxon */
+/* eslint no-undef: "error" */
+const { DateTime } = luxon.DateTime;
+
+const displayDate = () => {
+  const dateNowFormat = DateTime.now().toFormat('FF');
+  document.querySelector('.date-now').innerText = dateNowFormat;
+};
```
Comment (DeliceLydia) [inline app.js]: - [x] The current date is supposed to be displayed below the navbar and you should use `Luxon Library` I can see that you tried to use it below but it is not working now please fix it

File: index.html [hunk 2]
```diff
@@ -1,39 +1,83 @@
...
+        <ul class="contact__info">
+          <li class="email">Our email</li>
+          <li class="phone">Our phone number</li>
+          <li class="address">Our address</li>
+        </ul>
+      </div>
+    </main>
+    <footer class="footer">
+      <p>Copyright 2021</p>
+    </footer>
```
Comment (DeliceLydia) [inline index.html:79]: - [x] Footer has to be to the end of the page but currently when  you open the page it looks like this
footer

Please fix that!

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! However, there are some issues that you still need to work on to go to the next project but you are almost there! these are major highlights for me:

- PR is professional
- Commits message are descriptive


## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi @the-catalystmc and @arthurborgesdev , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Alt Text
 ## Required Changes ♻️
 - [ ] As per project requirements we have to follow the correct setup of the linter. I am not able to see `.stylelintrc.json` as well as `.eslintrc.json` file. Kindly follow the [link](https://github.com/microverseinc/linters-config/tree/master/html-css-js) for the correct setup of styling and eslint.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] Please try to add `live link` and screenshot in your `README` file
- [ ] Try to add some information in your `contact` section

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by arthurborgesdev** [comment]
@hemant-soni-vst-au4, The linters are inside the .gitignore file (following best practices), that's why you couldn't find them. You have to set up them by yourself.

**Comment by the-catalystmc** [comment]
hemant-soni-vst-au4 your explanation is poorly written and it does not make much sense. The project requirement states "Make sure that there are no linter errors." Those linters are run locally on my computer so I git ignore them as you can see on my .gitignore file.

**Comment by hemantso** [comment]
- My bad guys. I am just following the guidelines that there should be linter files. your repo passed all the checks. But why did you guys put these files in gitignore? 
- let's do one thing, can you guys raise one more CR request without putting the file in gitignore or without any change? make sense.

**Comment by SpaYco** [comment]
Hello @arthurborgesdev and @the-catalystmc,

This is an additional review as a result of the second opinion request.

You're right, those files don't need to be in the repo, as `.github/workflows/linters.yml` uses the files from Mcroverse's linters-config repo to run the GitHub Actions, which makes it the only file needed for the GitHub actions, but many people fail to realize that since they don't read the code inside the file.

We'll make sure to let all code reviewers know about this moving forward, sorry about the inconvenience.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Review by umararshadbut — APPROVED** [review]
Hi @the-catalystmc , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
HAPPY CODING :computer:
**Umair Arshad**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Pull request merged**